lode-viewer/application.js

```diff
@@ -42,7 +42,7 @@ export default class LodeApp extends Templated {
 	 */
 	Template() {
 		return  "<div handle='presentation' class='instructions'>nls(Map_Presentation_1)</div>" + 
-				"<div handle='presentation' class='instructions'>nls(Map_Presentation_2)</div>" + 
+				//"<div handle='presentation' class='instructions'>nls(Map_Presentation_2)</div>" + 
```

Comment:
Instead of commenting out this line it may be better to remove it to keep the code clean (especially if you know the div will never be included in future versions).

Reply:
I left it commented for now just in case someone suggests keeping an additional line of text. But yes, we should delete it afterwards.